internal/pkg/mount/pseudo.go

```diff
@@ -26,6 +26,7 @@ func PseudoSubMountPoints() (mountpoints *Points, err error) {
 	pseudo := NewMountPoints()
 	pseudo.Set("devshm", NewMountPoint("tmpfs", "/dev/shm", "tmpfs", unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_NODEV|unix.MS_RELATIME, ""))
 	pseudo.Set("devpts", NewMountPoint("devpts", "/dev/pts", "devpts", unix.MS_NOSUID|unix.MS_NOEXEC, "ptmxmode=000,mode=620,gid=5"))
+	pseudo.Set("hugetlb", NewMountPoint("none", "/dev/hugepages", "hugetlbfs", 0, ""))
```

Comment:
Should the source be "hugetlbfs" here instead of "none", or is that taken care of by the kernel?

Reply:
Good question. As far as I can tell `none` should work, and the fstype is the important bit. Still needs testing though.